storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst

```diff
@@ -240,11 +324,42 @@ Options
    Use this number of threads to rebuild indexes in a compact backup. Only has
    effect with --prepare and --rebuild-indexes.
 
+.. option:: --remove-original
+
+   Implemented in |Percona XtraBackup| 2.3.7, this option when specified will
+   remove :file:`.qp`, :file:`.xbcrypt` and :file:`.qp.xbcrypt` files after
+   decryption and decompression.
+
+.. option:: --safe-slave-backup
+
+   When specified, xtrabackup will stop the slave SQL thread just before
+   running ``FLUSH TABLES WITH READ LOCK`` and wait to start backup until
+   ``Slave_open_temp_tables`` in ``SHOW STATUS`` is zero. If there are no open
+   temporary tables, the backup will take place, otherwise the SQL thread will
+   be started and stopped until there are no open temporary tables. The backup
+   will fail if ``Slave_open_temp_tables`` does not become zero after
+   :option:`xtrabackup --safe-slave-backup-timeout` seconds. The slave SQL
+   thread will be restarted when the backup finishes.
+
+.. option:: --safe-slave-backup-timeout=SECONDS
+
+   How many seconds :option:`xtrabackup --safe-slave-backup` should wait for
+   ``Slave_open_temp_tables`` to become zero. Defaults to 300 seconds.
+
 .. option:: --secure-auth
 
    Refuse client connecting to server if it uses old (pre-4.1.1) protocol.
    (Enabled by default; use --skip-secure-auth to disable.)
 
+.. option:: --slave-info
+
+   This option is useful when backing up a replication slave server. It prints
```

Comment:
I am getting questions about this option sometimes. Description tells that "it prints the binary log position and name of the master server" and then "writes this information to the file". However, the name of the master server doesn't get to the file.